File: magicdb-server/magicdb-data-service/magicdb-data-service-web/src/main/java/ai/magicdb/dataservice/web/controller/SimpleDataServiceController.java
package ai.magicdb.dataservice.web.controller;

import ai.magicdb.server.tools.base.wrapper.result.DataResult;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

/**
 * 简单数据服务控制器
 * 用于处理 /api/data-service 路径的请求
 * 
 * @author magicdb
 */
@RestController
@RequestMapping("/api/data-service")
public class SimpleDataServiceController {

    /**
     * 获取数据服务信息
     */
    @GetMapping
    public DataResult<Map<String, Object>> getDataServiceInfo() {
        Map<String, Object> info = new HashMap<>();
        info.put("status", "active");
        info.put("version", "1.0.0");
        info.put("message", "Data service is working correctly");
        
        return DataResult.of(info);
    }
}

File: magicdb-server/magicdb-data-service/magicdb-data-service-web/src/main/kotlin/ai/magicdb/data/service/web/controller/SimpleDataServiceController.kt
package ai.magicdb.data.service.web.controller

import ai.magicdb.server.tools.base.wrapper.result.DataResult
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

/**
 * 简单数据服务控制器
 * 用于处理 /api/data-service 路径的请求
 * 
 * @author magicdb
 */
@RestController
@RequestMapping("/api/data-service")
class SimpleDataServiceController {

    /**
     * 获取数据服务信息
     */
    @GetMapping
    fun getDataServiceInfo(): DataResult<Map<String, Any>> {
        val info = mapOf(
            "status" to "active",
            "version" to "1.0.0",
            "message" to "Data service is working correctly"
        )
        
        return DataResult.of(info)
    }
}
